manifests/base.pp

```diff
@@ -85,11 +99,4 @@
       require => $_require,
     }
   }
-
-  if $rsyslog::manage_service {
```

Comment:
Is it just me, or does anyone else think the merging of parameters being done is overly complicated?

ie what's wrong with?
```suggestion
  if $rsyslog::manage_service {
    service { $rsyslog::service_name:
      ensure => $rsyslog::service_status,
      enable => $rsyslog::service_enabled,
    }
    if $rsyslog::manage_confdir {
      File[$rsyslog::confdir] ~> Service[$rsyslog::service_name]
    }
  }
```